src/test/groovy/com/intuit/graphql/orchestrator/integration/GoalServiceSpec.groovy

```diff
@@ -0,0 +1,69 @@
+package com.intuit.graphql.orchestrator.integration
+
+import com.google.common.collect.ImmutableMap
+import com.intuit.graphql.orchestrator.GoalsService
+import com.intuit.graphql.orchestrator.TestCase
+
+import helpers.BaseIntegrationTestSpecification
+
+class GoalServiceSpec extends BaseIntegrationTestSpecification {
+
+    void canQueryGoalService() {
+        given:
+        TestCase testCase = TestCase.newTestCase()
```

Comment:
We should convert this to like  how `QueryDirectiveSpec` was written.  So that we can add validation of execution input for the downstream.